src/docs/reference/modules/Migrations/README.md

```diff
@@ -42,7 +42,7 @@ namespace Members
 
         public int UpdateFrom1()
```

Comment:
```suggestion
        public async Task<int> UpdateFrom1Async()
```

Reply:
We would have to also update the calls to async if we change the signature. I'll update that in the AM

Reply:
We already can use both `UpdateFromN()` and `UpdateFromNAsync()`.

Reply:
I don't really understand; the caller is OC here, and OC supports *Async update methods. You changed the sample to use the async `SchemaBuilder` methods, so the method needs to be async too, because the current sample won't compile.

Reply:
Yes should use the async method, the automatic data migration already detect if it has to call a sync or async method

Reply:
You should using Async if your migration step does asynchronous call otherwise no need. In this PR, we are using all async methods to alter/create tables, so in this case we using the Async methods.  In OC both UpdateFrom1 and UpdateFrom1Async handle the same step. I think we first check if Async method exists and call it so it wins if both are defined in the same file

Reply:
No problem but now in the sample async call are done

Reply:
The following provided as a sample is wrong ;)

        public int UpdateFrom1()
        {
            await SchemaBuilder.CreateMapIndexTableAsync<MemberIndex>(table =>
            {